project.xml

```diff
@@ -50,7 +54,7 @@
         <option name = "use_checkout_explicit" value = "1" />
     </target>
 
-    <classfilename keep-tree = "true" pkgincludedir = "true" use-cxx = "true" pretty-print = "no" source-extension = "cc" />
+    <classfilename use-cxx-gcc-4-9 = "true" keep-tree = "true" pkgincludedir = "true" use-cxx = "true" pretty-print = "no" source-extension = "cc" />
```

Comment:
I don't see uses of `std::regex` in this codebase; do we require (or soon plan to require) gcc-4.9 for `fty-rest`? So far it was ok with default 4.8 I believe.